runtime/src/master.rs

```diff
@@ -186,16 +196,15 @@ decl_module! {
             proposal: Box<<T as Trait>::Call>,
```

Comment:
I don't think there is ever a reason for a submitter to run the normal `execute()` economically it's always in the submitter's best interest to use `execute_unchecked_wieght` with weight set to 0. I guess it doesn't mater in the long run because Master with go away later.